lively.merger/merger.js

```diff
@@ -42,10 +37,11 @@ export class Merger {
     let propertiesMorph1 = this.propertiesFromMorph(morph1);
     let propertiesMorph2 = this.propertiesFromMorph(morph2);
 
-    let properties = merge(
+    let result = mergeObjects(
       propertiesBaseMorph,
       propertiesMorph1,
       propertiesMorph2);
-    return new Morph(properties);
+    console.log(result.mergeConflicts); // TODO conflict resolve
```

Comment:
I  don't really like console.log in "production code". The TODO comment is enough to find this again.

```suggestion
    // TODO conflict resolve
```